Source/Core/DolphinQt/GameList/GameFile.cpp

```diff
@@ -203,8 +226,7 @@ void GameFile::SaveToCache()
 	stream.setVersion(DATASTREAM_REVISION);
 	stream << CACHE_REVISION;
 
-	stream << m_folder_name
```

Comment:
Ohh, because we need it for the game list. I hope the overhead of computing it every time is not too noticeable...